package.json

```diff
@@ -41,6 +41,7 @@
   "homepage": "https://github.com/buildo/react-components",
   "dependencies": {
     "@types/lodash": "4.14.74",
+    "@types/node": "^8.0.24",
```

Comment:
why? BRC is a browser lib, why should we need these?
if they're for tests please move to dev-deps

also, **remove the ^**!!1!11!!

Reply:
I think this is for `process.env`

Reply:
but I wouldn't use `@types/node` any way since this is not node, but `process.env` is inserted by webpack. I would just follow the approach suggested here: https://medium.com/the-missing-bit/using-node-env-with-typescript-and-webpack-689a85041978

Reply:
but using `const` instead of `var` :P

Reply:
I `declare`d it in `utils/log.ts`
we can `declare` it globally

Reply:
@FrancescoCioria you shoul remove the `^` too: https://github.com/buildo/react-components/blame/master/package.json#L74 !11!!!!!